cli/src/main/java/org/jboss/as/cli/gui/OperationDialog.java

```diff
@@ -328,6 +329,10 @@ public RequestProp(String name, ModelNode props, ModelNode value) {
                 this.nillable = props.get("nillable").asBoolean();
             }
 
+            if (props.get("expressions-allowed").isDefined()) {
```

Comment:
Use

```
props.hasDefined("expressions-allowed")
```

The ModeNode "get" method actually mutates the model node by adding an undefined child under that key. This makes a fluent API like node.get("enabled").set(true) possible. But it also means we need to use "hasDefined" for this kind of check to avoid unintended mutation of the node. In this case it probably doesn't matter, but we enforce this as a code style kind of thing so people always see the correct usage and thus use it when it really does matter.